### PR TITLE
Added some basic additional commands for few applications

### PR DESCRIPTION
**Pull request by SandeepJoel**
Added commands for slack, vscode and git applications

### CODE REVIEW

**Review by rntz — APPROVED** [review]
lgtm

File: apps/vscode/vscode.talon [hunk 1]
```diff
@@ -69,6 +70,12 @@ file rename:
 file move:
     user.vscode("fileutils.moveFile")
     sleep(150ms)
+file duplicate:
```
Comment (pokey) [inline apps/vscode/vscode.talon:73]: I use "clone" instead as it's shorter and seems to recognise well
```suggestion
file clone:
```

**Review by pokey — COMMENTED** [review]
Looks good with one minor tweak I might make

**Review by pokey — APPROVED** [review]
Looks good!

**Pull request merged**